juriscraper/opinions/united_states/state/ind.py

```diff
@@ -24,12 +25,48 @@ def _process_html(self):
         for case in self.html:
             if case["courtName"] != self.court_name:
                 continue
+
+            is_percuriam = case["opinion"]["perCuriam"]
+            other_courts = [
+                c for c in case["courts"] if c["name"] != self.court_name
+            ]
+
             self.cases.append(
                 {
                     "name": case["style"],
                     "docket": case["caseNumber"],
                     "date": case["date"],
                     "url": f"https://public.courts.in.gov/Decisions/{case['opinionUrl']}",
                     "disposition": case["decision"],
+                    "lower_court": ", ".join(c["name"] for c in other_courts),
+                    "lower_court_number": ", ".join(
+                        c["number"] for c in other_courts
+                    ),
+                    "judge": ", ".join(
+                        self.clean_judge_name(v["judge"])
+                        for v in case["opinion"]["votes"]
+                    ),
+                    "author": ""
+                    if is_percuriam
+                    else case["opinionText"]
+                    .replace("in an opinion by ", "")
+                    .replace(".", ""),
+                    "per_curiam": is_percuriam,
+                    "type": case["opinion"]["result"],
```

Comment:
can we move the author stuff out of one line territory - its pretty hard to read.